README.md

```diff
@@ -49,6 +49,34 @@ Resources
 - [Code Examples](examples)
 - [API reference](https://odlgroup.github.io/odl/odl.html)
 
+Applications
+============
+ODL has been used in a wide range of research and development with several research articles and conference contributions using ODL, as well as other open source projects using ODL as a back-end. Many of these also release their code publicly can serve as examples of how to use ODL. This is a short list of such projects. If you want your project added, contact the developers.
+
+##### Articles
+
+
+| Description      |  Code  |
+|------------------|--------|
+| *High-level algorithm prototyping: an example extending the TVR-DART algorithm*. Conference on Discrete Geometry for Computer Imagery (DGCI), 2017 | [GitHub](https://github.com/aringh/TVR-DART) |
+| *GPUMCI, a ﬂexible platform for x-ray imaging on the GPU*. Fully3D, 2017 | N/A |
+| *Generalized Sinkhorn iterations for regularizing inverse problems using optimal mass transport*. [Arxiv](https://arxiv.org/abs/1612.02273) |  N/A |
+| *Spectral CT reconstruction with anti-correlated noise model and joint prior*. Fully3D, 2017 | [GitHub](https://github.com/adler-j/spectral_ct_examples) |
+| *Solving ill-posed inverse problems using iterative deep neural networks*. [Arxiv](https://arxiv.org/abs/1704.04058) | [GitHub](https://github.com/adler-j/learned_gradient_tomography) | 
+| *A modified fuzzy C means algorithm for shading correction in craniofacial CBCT images*. CMBEBIH, 2017 | [GitHub](https://github.com/adler-j/mfcm_article) | 
+| *Total variation regularization with variable Lebesgue prior*. [Arxiv](https://arxiv.org/abs/1702.08807) | [GitHub](https://github.com/kohr-h/variable_lp_paper) |
```

Comment:
I agree, will get that fixed. Currently ordering is approximately by answers to the issue :O